doc/source/citation.md

```diff
@@ -0,0 +1,114 @@
+(citation)=
+
+# Citing and method overview
+
+When using a method implemented in xDEM, one should cite both the package and the original study behind the method (if there is any)!
+
+## Citing xDEM
+
+To cite the package, use the Zenodo DOI: [![Zenodo](https://zenodo.org/badge/doi/10.5281/zenodo.4809697.svg)](https://zenodo.org/doi/10.5281/zenodo.4809697).
```

Comment:
I think we need to be more detailed than this. The best way is to have a ref like "(xDEM contributors, 202X)" and in the reference list: "Xdem Contributors: xdem, Zenodo [code], https://doi.org/10.5281/zenodo.YYYY, 202X." with year and zenodo link to updated depending on the version and year used.

Reply:
It will be easier when we have a proper article for it...

Reply:
The link I put is the generic Zenodo link that works for all versions.
I wanted to add a comment about the option to point to a specific version, but we haven't done this so well until now... (We only have 0.0.2 and 0.0.18, which is a bit weird).

@erikmannerfelt Can you activate user sharing on the Zenodo for @adehecq and me (https://help.zenodo.org/docs/share/user-sharing/), so that any one of us can update the versioning when others are too busy. Or maybe we can even set-up auto-updates on version + authors? I've seen several packages do it.

Reply:
My bad! I forgot you added us, and we can edit when logged in Zenodo with our ORCID. All good!